src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/OverrideEqualsOnOverloadingOperatorEqualsTests.cs

```diff
@@ -17,7 +18,163 @@ protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpOverrideEqualsOnOverloadingOperatorEqualsAnalyzer();
+            throw new NotSupportedException("CA2224 is not applied to C# since it already reports CS0660");
```

Comment:
I'm not sure what you're asking, but I suspect your query is the same as in OverrideEqualsOnOverloadingOperatorEqualsTests.Fixer.cs, in which case, the answer is the same.